pipelines/nextflow/subworkflows/additional_seq_prepare.nf

```diff
@@ -36,25 +37,34 @@ workflow additional_seq_prepare {
         gb_file = DOWNLOAD_GENBANK(accession)
 
         // Parse data from GB file into GFF3 and json files
-        EXTRACT_FROM_GB(prefix, production_name, gb_file)
-        PROCESS_GFF3(EXTRACT_FROM_GB.out.gene_gff, EXTRACT_FROM_GB.out.genome, accession)
-        GFF3_VALIDATION(PROCESS_GFF3.out.gene_models)
+        EXTRACT_FROM_GB(DOWNLOAD_GENBANK.out.downloaded_gb_data, prefix, production_name)
+
+        PROCESS_GFF3(EXTRACT_FROM_GB.out.gene_gff, EXTRACT_FROM_GB.out.genome)
+
+        // Tidy and validate gff3 using gff3validator (NOTE: Requires `module load libffi-3.3-gcc-9.3.0-cgokng6`)
+        GFF3_VALIDATION(PROCESS_GFF3.out.functional_annotation, PROCESS_GFF3.out.gene_models)
```

Comment:
GFF3_VALIDATION step fails for me as it expects only 1 input, so we need to validate  `PROCESS_GFF3.out.functional_annotation` ? I am not sure

Reply:
Same issue is changes to gff_validation.nf are not in this PR sorry....  let me update with incoming PR and it will work :)